File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    answer = []
    R = len(wallpaper)
    C = len(wallpaper[0])
    M_x = -1
    m_x = C
    M_y = -1
    m_y = R
    for i in range(R):
        for j in range(C):
            if wallpaper[i][j] == '#':
                if i > M_y:
                    M_y = i
                if j > M_x:
                    M_x = j
                if i < m_y:
                    m_y = i
                if j < m_x:
                    m_x = j
    
            
    return [m_y, m_x, M_y+1, M_x+1]

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(vector<string> wallpaper) {
    vector<int> answer;
    int small_y=51;
    int large_y=0;
    int small_x=51;
    int large_x=0;
    for(int i=0;i<wallpaper.size();i++){
        for(int j=0;j<wallpaper[i].size();j++){
            if(wallpaper[i][j]=='#'){
                cout << i<<" "<<j;
                if(i<small_y){
                    small_y = i;
                }
                if(i+1>large_y){
                    large_y = i+1;
                }
                if(j<small_x){
                    small_x = j;
                }
                if(j+1>large_x){
                    large_x = j+1;
                }
            }
        }
    }
    answer = {small_y, small_x, large_y,large_x};
    return answer;
}